### DISCUSSION TITLE
Merge Newpipe v0.26.0

### DISCUSSION THREAD

**Discussion by mikitakahazekura** [TeamNewPipe/NewPipe#10686] [Questions] [answered]
As the title suggest, merge new functionalities from Newpipe v0.26.0

**Comment by opusforlife2** [comment]
@mikitakahazekura ?

**Reply by TobiGr** [reply, marked as answer]
This was most likely meant to be opened in a fork

**Reply by mikitakahazekura** [reply]
Sorry my bad, it can be closed.
